rpa-manager/modules/ROOT/pages/processautomation-deploy.adoc

```diff
@@ -178,10 +180,21 @@ When creating new schedules, you should pay attention to whether the new schedul
 Click on the button *Add Schedule* to assign a (further) schedule to this configuration and complete the form *Create a new schedule*:
 +
 include::rpa-manager::partial$form-scheduling.adoc[]
+* *Bots*:
++
+Select all RPA Bots that are to carry out the production configuration.
++
+Select the checkbox *Show bots not enabled in this phase* to also show bots that are not intended for use in this phase.
++
+When you select a bot, ensure that it is configured for the required session type, is allowed to be deployed in the relevant phase, and that it can operate the required applications.
```

Comment:
Try to be consistent with the structure:
...ensure that it is configured... that it is allowed... and that it can operate... 
or
...ensure that it is configured... is allowed... and can operate...